python/pyspark/sql/connect/plan.py

```diff
@@ -642,7 +664,7 @@ def _repr_html_(self) -> str:
         return f"""
         <ul>
             <li>
-                <b>Union</b><br />
+                <b>SetOperation</b><br />
```

Comment:
qq: why don't we have Except, Union, etc separately?

Reply:
@HyukjinKwon I hope this discussion can answer your question https://github.com/apache/spark/pull/38166#discussion_r991783272....